lib/block_view.go

```diff
@@ -117,6 +118,11 @@ type UtxoView struct {
 
 	// Validator mappings
 	ValidatorPKIDToValidatorEntry map[PKID]*ValidatorEntry
+	// Note: I really don't like using a string for the map key here,
+	// as it loses the intent. However, bls.PublicKey does not implement
+	// the comparable interface for golang maps and implementing it
+	// would require greater changes to the BLS package, which isn't worth it.
```

Comment:
Feel free to remove this comment, now that we have a well encapsulated serialized map key type for the BLS public key
```suggestion
```